### PR TITLE
fix hvdc modification and creation logs

### PR DESCRIPTION
**Pull request by EtienneLt**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/VscModification.java [hunk 1]
```diff
@@ -209,46 +240,59 @@ private void hvdcAngleDroopActivePowerControlAdder(HvdcLine hvdcLine, ReportNode
...
+            .withUntypedValue("id", converterStation.getId())
+            .withSeverity(TypedValue.INFO_SEVERITY)
+            .add();
+
+        // characteristic
+        List<ReportNode> characteristicReports = new ArrayList<>();
         if (converterStationModificationInfos.getEquipmentName() != null && converterStationModificationInfos.getEquipmentName().getValue() != null) {
-            ModificationUtils.getInstance().applyElementaryModifications(converterStation::setName, () -> converterStation.getOptionalName().orElse("No value"), converterStationModificationInfos.getEquipmentName(), converterStationReportNode, "Name");
+            characteristicReports.add(ModificationUtils.getInstance().applyAndBuildModificationReport(converterStation::setName,
+                () -> converterStation.getOptionalName().orElse("No value"), converterStationModificationInfos.getEquipmentName(), "Name"));
```
Comment (Mathieu-Deharbe) [inline src/main/java/org/gridsuite/modification/server/modifications/VscModification.java:261]:
```suggestion
                () -> converterStation.getOptionalName().orElse(NO_VALUE), converterStationModificationInfos.getEquipmentName(), "Name"));
```

File: src/main/java/org/gridsuite/modification/server/modifications/VscModification.java [hunk 2]
```diff
@@ -209,46 +240,59 @@ private void hvdcAngleDroopActivePowerControlAdder(HvdcLine hvdcLine, ReportNode
...
         }
 
         if (converterStationModificationInfos.getVoltageSetpoint() != null) {
-            ModificationUtils.getInstance().applyElementaryModifications(converterStation::setVoltageSetpoint, converterStation::getVoltageSetpoint, converterStationModificationInfos.getVoltageSetpoint(), converterStationReportNode, "Voltage");
+            setPointsReports.add(ModificationUtils.getInstance().applyAndBuildModificationReport(converterStation::setVoltageSetpoint,
+                converterStation::getVoltageSetpoint, converterStationModificationInfos.getVoltageSetpoint(), "Voltage"));
+        }
+        if (!setPointsReports.isEmpty()) {
+            ModificationUtils.getInstance().reportModifications(converterStationReportNode,
+                setPointsReports, "SetPoints", "SetPoints", Map.of());
```
Comment (Mathieu-Deharbe) [inline src/main/java/org/gridsuite/modification/server/modifications/VscModification.java:292]:
```suggestion
                setPointsReports, SETPOINTS, SETPOINTS, Map.of());
```
One less upper case 'P' in `SETPOINTS` though. But I guess it is not important ?

**Review by Mathieu-Deharbe — APPROVED** [review]
Tested. Only minor suggestions, do as you wish.

File: src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java [hunk 3]
```diff
@@ -173,10 +174,8 @@ private VscConverterStation createConverterStation(Network network,
         VscConverterStation vscConverterStation = voltageLevel.getTopologyKind() == TopologyKind.NODE_BREAKER ?
                 createConverterStationInNodeBreaker(network, voltageLevel, converterStationCreationInfos, converterStationReporter) :
                 createConverterStationInBusBreaker(voltageLevel, converterStationCreationInfos, converterStationReporter);
-
         converterStationReporter.newReportNode()
-                .withMessageTemplate("converterStationCreated" + logFieldName, "New converter station with id=${id} created")
-                .withUntypedValue("id", converterStationCreationInfos.getEquipmentId())
+                .withMessageTemplate("converterStationCreated" + logFieldName, "New converter station " + converterStationCreationInfos.getEquipmentId() + " created")
```
[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java:178]: Well I dont know why but the old way does not work it gives 

New converter station with id=${id} created

and ${id} is not replaced

[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java:178]: so I fixed it

File: src/main/java/org/gridsuite/modification/server/modifications/VscModification.java [hunk 4]
```diff
@@ -70,34 +67,66 @@ private void modifyVsc(@NonNull Network network, @NonNull HvdcLine hvdcLine, Vsc
...
+        characteristics(hvdcLine, modificationInfos, subReportNode);
+
+        // Set Points
+        //  Set Points
+        List<ReportNode> setPointsReports = setPoints(hvdcLine, modificationInfos);
+        //  hvdc droop
+        List<ReportNode> droopReports = hvdcAngleDroopActivePowerControlAdder(hvdcLine);
+
+        if (!setPointsReports.isEmpty() || !droopReports.isEmpty()) {
+            ReportNode setPointsReport = subReportNode.newReportNode().withMessageTemplate(VSC_SETPOINTS, SETPOINTS).add();
```
[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/main/java/org/gridsuite/modification/server/modifications/VscModification.java:80]: I agree I'll ask the PO

[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/main/java/org/gridsuite/modification/server/modifications/VscModification.java:80]: For now we keep setPoints we'll see later

**Review by Mathieu-Deharbe — APPROVED** [review]

File: src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mathieu-Deharbe) [reply src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java:176]: I tried with the original code and it worked fine for me.

Also, please don't close the conversations that are not finished.

[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java:176]: ok sorry I reverse

**Pull request merged**